addons/mail/tools/mail_validation.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+
+from odoo import tools
+
+_logger = logging.getLogger(__name__)
+
+_flanker_lib_warning = False
+
+try:
+    from flanker.addresslib import address
+    # Avoid warning each time a mx server is not reachable by flanker
+    logging.getLogger("flanker.addresslib.validate").setLevel(logging.ERROR)
+
+    def mail_validate(email):
+        return bool(address.validate_address(email))
+
+except ImportError:
+
+    def mail_validate(email):
+        global _flanker_lib_warning
+        if not _flanker_lib_warning:
+            _logger.info("The `flanker` Python module is not installed, so email validation fallback to email_normalize. Use 'pip install flanker' to install it")
```

Comment:
Maybe "The `flanker` Python module is not installed. Email validation therefore falls back on simple email normalization check. Use 'pip install flanker' to install the missing library." ?

+ not sure, do we have to export string for 14.2 @mart-e ?

Reply:
> And not sure, do we have to export string for 14.2 @mart-e ?

I haven't moved https://www.transifex.com/odoo/odoo-master to saas-14.2 yet (still .1) but plan to do it this week. So you can but it's not a big deal if you don't.

Reply:
but you don't translate logger content, only for messages displayed to endusers, not for sysadmin

Reply:
Ah foert right. Stupid tde is stupid.